lib/gollum/views/history.rb

```diff
@@ -33,6 +34,13 @@ def versions
       def editable
         @editable
       end
+
+      private
+
+      def path_for_version(sha)
+        Gollum::PreviewPage.new(@wiki, @pathnames[sha], '', sha).escaped_url_path
```

Comment:
A bit annoying that we have to instantiate a new PreviewPage object for each revision. Alternative would be to refactor `gollum-lib` in such a way that the `escaped_url_path` method depends on a class method that we could also invoke here.

Reply:
I would prefer the second option. It seems wasteful to instantiate PreviewPage objects for what is basically a string manipulation operation.

Reply:
Ok, agreed!